File: legacy/src/main/java/com/redbox/domain/admin/dto/AdminDetailResponse.java
package com.redbox.domain.admin.dto;

import com.redbox.domain.attach.dto.AttachFileResponse;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.util.List;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class AdminDetailResponse {
    private Long id;
    private String title; // 제목
    private String author; // 작성자
    private LocalDate date; // 등록일
    private LocalDate startDate; // 기부 시작일
    private LocalDate endDate; // 기부 종료일
    private int targetAmount; // 목표 수량
    private String status;
    private int views; // 조회수
    private String content; // 내용
    private List<AttachFileResponse> attachFileResponses; // 첨부 파일 리스트
}

File: migration/src/main/kotlin/com/redbox/domain/community/funding/dto/AdminDetailResponse.kt
package com.redbox.domain.community.funding.dto

import com.redbox.domain.community.attach.dto.AttachFileResponse
import java.time.LocalDate

data class AdminDetailResponse(
    val id: Long,
    val title: String,
    val author: String,
    val date: LocalDate,
    val startDate: LocalDate,
    val endDate: LocalDate,
    val targetAmount: Int,
    val status: String,
    val views: Int,
    val content: String,
    val attachFiles: List<AttachFileResponse>
)